build-cwl-docker.sh

```diff
@@ -1,4 +1,3 @@
 #!/bin/sh
 set -e
-docker build --file=cwltool_module.Dockerfile --tag=commonworkflowlanguage/cwltool_module .
```

Comment:
Can you restore the cwltool_module Docker image?  Some things actually do rely on it.  Its purpose is to provide an environment for running schema salad and cwltool for operations that don't require docker.

Reply:
Would the new image with `sh` as an entrypoint work for this purpose?

Reply:
Yes, I think that would work.  The cwltool_module image is used to run "python -mschema_salad.makedoc", the key difference is that cwltool_module shouldn't have `WORKDIR /error` and `ENTRYPOINT`.  The simple solution is to layer the cwltool image on top of cwltool_module.

Reply:
Decided to build cwltool upon cwltool_module, just like it was before.

Let me know it this change is uncomfortable because some parameters used in the command that builds the cwltool_module images changed a bit.